src/components/BackstageComparison/FeatureComparisonTable.js

```diff
@@ -0,0 +1,413 @@
+import React, { Fragment } from 'react';
+import { CheckIcon, MinusIcon } from '@heroicons/react/solid';
+import { TextLink } from 'components';
+
+import COMPETITORS from './competitors';
+
+const sections = [
+  {
+    name: 'Software catalog',
+    features: [
+      {
+        name: 'Bring your own data model',
+        competitor: { Roadie: 'Yes', Backstage: 'Yes, with some effort' },
+      },
+      {
+        name: 'Entity definition',
+        competitor: { Roadie: 'Markdown, UI, or via API', Backstage: 'Markdown' },
+      },
+      {
+        name: 'API-based ingestion and update',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Out of the box data ingestion (k8s, github, CI/CD, etc)',
+        competitor: { Roadie: true, Backstage: true },
+      },
+      {
+        name: 'SCM Support',
+        competitor: {
+          Roadie:
+            'GitHub Cloud, GitHub Enterprise Server, Azure DevOps, GCP, GitLab Cloud, GitLab On-Prem, BitBucket Cloud, BitBucket Server',
+          Backstage:
+            'GitHub Cloud, GitHub Enterprise Server, Azure DevOps, GCP, GitLab Cloud, GitLab On-Prem, BitBucket Cloud, BitBucket Server, Gerrit',
+        },
+      },
+      {
+        name: 'Catalog UI customisation',
+        competitor: { Roadie: 'Yes', Backstage: 'Yes, with some effort' },
+      },
+    ],
+  },
+  {
+    name: 'Scorecards',
+    features: [
+      {
+        name: 'Extensible through the creation of custom data sources',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Operational reviews',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Levels',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Campaigns',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Team-based rollups',
+        competitor: { Roadie: true, Backstage: false },
+      },
+    ],
+  },
+  {
+    name: 'Scaffolder',
+    features: [
+      {
+        name: 'Self-serve actions',
+        competitor: { Roadie: true, Backstage: true },
+      },
+      {
+        name: 'Custom self-serve actions',
+        competitor: { Roadie: true, Backstage: true },
+      },
+      {
+        name: 'Custom field extensions',
+        competitor: { Roadie: true, Backstage: true },
+      },
+      {
+        name: 'Fallback and retry',
+        competitor: { Roadie: true, Backstage: true },
+      },
+    ],
+  },
+  {
+    name: 'TechDocs',
+    features: [
+      {
+        name: 'Docs creation engine',
+        competitor: { Roadie: 'TechDocs', Backstage: 'TechDocs' },
+      },
+      {
+        name: 'Extensibility via plugins',
+        competitor: { Roadie: true, Backstage: true },
```

Comment:
I think there's a benefit to add here about how we build docs in the background by default?

Reply:
Not sure how we'd phrase it, so I've asked on Slack. I'm ok to leave this out this time